### DISCUSSION TITLE
QTab - Add option for sending QIcon props.

### DISCUSSION THREAD

**Discussion by DouglasCalora** [quasarframework/quasar#14110] [Ideas / Proposals]
Today I can send only the icon I want in QTab, but if I want to change the style of the icon I can't, e.g., change the icon color but not the label color of QTab, or change the icon size and so on.

![image](https://user-images.githubusercontent.com/32134017/182179618-a878f9af-8b39-4a99-8686-18146cc05e9a.png)

**Comment by MilosPaunovic** [comment]
Couldn't you get by with the `content-class` prop?
https://codepen.io/milospaunovic/pen/PoREroG?editors=1010

**Reply by DouglasCalora** [reply]
No, this way changed the label color too, not only the icon color.

**Comment by MilosPaunovic** [comment]
In that case, CSS is your friend:
https://codepen.io/milospaunovic/pen/PoREroG
